src/utils/security/core.cljs

```diff
@@ -58,3 +59,10 @@
   and does not contain an rtlo character, which might mean that the url is spoofed"
   [text]
   (not (re-matches rtlo-link-regex text)))
+
+(defn hash-masked-password
```

Comment:
I think you can `comp` as well here since it's all function composition:
`(def hash-masked-password (comp masked-password safe-unmask-data native-module/sha3 mask-data))`

Reply:
love it